src/runtime/cache.ts

```diff
@@ -180,12 +181,15 @@ export function defineCachedEventHandler<T = any>(
       if (key) {
         return escapeKey(key);
       }
+      const varyParts =
+        opts.varies?.map((varyHeader) => event.node.req.headers[varyHeader]) ??
```

Comment:
This would be nice if keep them as pair like `['host_foo.com','x-region_asia']` to avoid conflicting same values with different header name.

Reply:
@pi0 Good points. Will change it.
Regarding the cache name, can we get problems with to long cache keys?

Reply:
Good point (used `:{name:hash(value}[]` in last impl)